src/content/api/module-methods.md

```diff
@@ -147,6 +149,8 @@ T> Note that all options can be combined like so `/* webpackMode: "lazy-once", w
 
 T> Note that `webpackInclude` and `webpackExclude` options do not interfere with the prefix. eg: `./locale`.
 
+`webpackExports`: Before webpack v5.0.0-beta.18, dynamic import would include the entire module even when you use a part of it. With `webpackExports` introduced, you can now request webpack to bundle only the used parts of module which would decrease the output chunk size.
```

Comment:
Some of this information is not necessary for the documentation reader, for example knowing what happened before a given beta version. This can be clearer in my opinion, please consider this:
```suggestion
`webpackExports`: tells webpack to only bundle the used exports of a module when using dynamic imports.  Can decrease the output size of a chunk. Available since webpack 5.0.0-beta.18
```

P.S. we dont write `v` in version: https://webpack.js.org/contribute/writers-guide/#typesetting